Sources/AblyAssetTrackingInternal/EnhancedLocationUpdateMessage.swift

```diff
@@ -0,0 +1,13 @@
+import AblyAssetTrackingCore
+
+public class EnhancedLocationUpdateMessage: Codable {
```

Comment:
why this is no struct type? It's recommended type for Codable objects.

Reply:
This is also code I did not write, but just moved because I needed to move them into a new module and also add the new `import`. I guess Github got confused. I really think a lot of classes should be structs in this project, but the main reason it did not use a single struct is because it was trying to support Objective-C, and using `@obj-c` or inheriting from NSObject.